enemy-data/abyss-order/abyss-mage.md

```diff
@@ -106,7 +111,10 @@ Stories circulate among adventurers that Abyss Mages are at the lowest rung of t
 
 ## Attacks
 
-> todo
+* Hurls a water bubble towards the player.
+* Teleports closer to the player, the teleport can hurt the player if close enough.
+* Creates a homing bubble that will slowly follow the player until either being popped, entraping a player or an enemy. If the bubble comes in contact with an entity, it will trap them for a period of time, rendering them unnable to do anything.
```

Comment:
Perhaps describe "popping", like "unless attacked" or something
"unnable" should be unable